src/main/java/by/onliner/page/AboutCompanyPage.java

```diff
@@ -0,0 +1,43 @@
+package by.onliner.page;
+
+import by.onliner.core.elements.widget.Text;
+import by.onliner.core.utils.Waiter;
+import io.qameta.allure.Step;
+import org.openqa.selenium.support.FindBy;
+import static by.onliner.constants.OnlinerConstants.ABOUT_LINK;
+
+public class AboutCompanyPage extends AbstractPage{
+
+    @FindBy(xpath = "//div[@class='news-header__title']/h1")
+    private Text pageMessage;
+
+    public AboutCompanyPage() {
+        super();
+    }
+
+    @Override
+    protected void waitForPageOpened() {
+        Waiter.waitForVisibility(pageMessage);
+    }
+
+    /***
+     * Read current url of page
```

Comment:
Update JavaDock, currently it doesn't make sense

Reply:
Updated